.github/workflows/au4_testflow.yml

```
@@ -0,0 +1,107 @@
+    - name: Report disabled test case
+      if: matrix.status == 'DISABLED'
+      env:
+        SCRIPT: ${{ matrix.script }}
+      run: echo "::warning title=Test case disabled::$SCRIPT is disabled - see share/testflowscripts/disabled.json for the reason"
+
+    # Before the artifact download: actions/checkout cleans the workspace
+    - name: Clone repository
+      if: matrix.status != 'DISABLED'
+      uses: actions/checkout@v6
+      with:
+        repository: ${{ inputs.app_repo || job.workflow_repository }}
```

actionlint (1.7.12)

[error] property "workflow_repository" is not defined in object type {check_run_id: number; container: {id: string; network: string}; services: {string => {id: string; network: string; ports: {string => string}}}; status: string}

(expression)